src/wallet/test/wallet_tests.cpp

```diff
@@ -47,14 +47,27 @@ BOOST_FIXTURE_TEST_CASE(rescan, TestChain100Setup)
 
     auto locked_chain = chain->lock();
 
+    // Verify ScanForWalletTransactions accomodates a null start block.
```

Comment:
This seems to trigger a lint warning from `test/lint/lint-all.sh` on master:

```
src/wallet/test/wallet_tests.cpp:50: accomodates  ==> accommodates
^ Warning: codespell identified likely spelling errors. Any false positives? Add them to the list of ignored words in test/lint/lint-spelling.ignore-words.txt
```

Reply:
Should be fixed in #15102.